core/admin/mailu/utils.py

```diff
@@ -498,3 +476,24 @@ def cleaner():
 
 cleaned = Value('i', False)
 session = MailuSessionExtension()
+
+# this is used by the webmail to authenticate IMAP/SMTP
+def verify_temp_token(email, token):
+    try:
+        if token.startswith('token-'):
+            sessid = app.session_store.get(token)
+            if sessid:
```

Comment:
can be written as
`if sessid := app.session_store.get(token):`

Reply:
That's c161a2c9